app/assets/stylesheets/about.scss

```diff
@@ -19,8 +20,13 @@
   width: 100%;
 }
 
+.about .the-board .card.blank {
```

Comment:
Apparently my editor wanted to format some things here. This is the only thing that really changed in this file - I added this class.